server/manager/src/main/java/org/apache/accumulo/manager/compaction/queue/CompactionJobPriorityQueue.java

```diff
@@ -304,4 +309,8 @@ private CjpqKey addJobToQueue(TabletMetadata tabletMetadata, CompactionJob job)
     jobQueue.put(key, new CompactionJobQueues.MetaJob(job, tabletMetadata));
     return key;
   }
+
+  public synchronized void clear() {
+    jobQueue.clear();
```

Comment:
tabletJobs needs to be cleared also, its like an index for jobQueue that helps quickly find what a tablet has queued

```suggestion
    jobQueue.clear();
    tabletJobs.clear();
```

Reply:
Cleared tabletJobs in 18d2243